frontend/src/components/LendingForm.js

```diff
@@ -57,30 +58,36 @@ const LendingForm = ({walletId}) => {
             const backendUrl = process.env.BACKEND_URL || 'http://127.0.0.1:8000';
             console.log("BACKENDURL", backendUrl)// Replace with your backend URL
             console.log("Query Params:", queryParams);
-            const response = await fetch(`${backendUrl}/transaction-data?${queryParams}`, {
-                method: 'GET',
+            // const response = await fetch(`${backendUrl}/transaction-data?${queryParams}`, {
+            //     method: 'GET',
+            //     headers: {
+            //         'Content-Type': 'application/json',
+            //     },
+            // });
+
+            const response = await axios.get(`${backendUrl}/transaction-data`, {
+                params: queryParams,
                 headers: {
                     'Content-Type': 'application/json',
                 },
             });
 
             console.log("Response:", response);
-            if (response.ok) {
-                const data = await response.json();
-                console.log(data);
-                setTransactionData(data);
-                console.log("Transaction data fetched successfully:", data);
+            if (response.status === 200) {
+                console.log(response);
+                setTransactionData(response);
+                console.log("Transaction data fetched successfully:", response);
 
                 try {
-                    const txResult = await sendTransaction(data);
+                    const txResult = await sendTransaction(response);
```

Comment:
did you make sure it works correctly after the change? it seems like you are passing whole response object to `sendTranscation` instead of response.data